.github/workflows/validate-other-market-data.yml

```diff
@@ -48,6 +48,13 @@ jobs:
           make build
           echo "Starting localnet..."
           DOCKER_BUILDKIT=1 make localnet-startd
+
+      - name: Get diff of otherMarketData.json
+        run: |
+          git fetch origin
+          git diff remotes/origin/main -- public/configs/otherMarketData.json > otherMarketDiff.txt
```

Comment:
is there any cleanup of otherMarketDiff after?

Reply:
i think it's just part of the workflow. so when workflow finishes, everything is cleaned up